docs/release-notes/8.3.asciidoc

```diff
@@ -5,6 +5,11 @@
 [[release-notes-8.3.3]]
 === 8.3.3
 
+[discrete]
+[[known-issue-8.3.3]]
+==== Known issues
+* An {endpoint-cloud-sec} bug on macOS and Linux can cause CPU spikes if malware protection is enabled on an {endpoint-cloud-sec} integration policy. When this happens, {endpoint-cloud-sec} may experience system coverage gaps. To avoid this, we recommend using {agent} version 8.3.2 or earlier. If you are using {agent} version 8.3.3 and have encountered this issue, you can temporarily resolve it by rebooting your computer and disabling <<<<malware-protection, malware protection>> on your {endpoint-cloud-sec} integration policy (https://github.com/elastic/endpoint/issues/22[#22]).
```

Comment:
```suggestion
* An {endpoint-cloud-sec} bug on macOS and Linux can cause CPU spikes if malware protection is enabled on an {endpoint-cloud-sec} integration policy. When this happens, {endpoint-cloud-sec} may experience system coverage gaps. To avoid this, we recommend using {agent} version 8.3.2 or earlier. If you are using {agent} version 8.3.3 and have encountered this issue, you can temporarily resolve it by rebooting your computer and disabling <<malware-protection, malware protection>> on your {endpoint-cloud-sec} integration policy (https://github.com/elastic/endpoint/issues/22[#22]).
```